english/car/regera.txt

```diff
@@ -0,0 +1 @@
+An hybrid supercar by Koenigsegg. 0-400km/h in 20 holy seconds! and a top speed of 410. Four cars sold before anybody saw the car.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.